source/connect-atlas-cli.txt

```diff
@@ -113,7 +113,17 @@ from the {+atlas-cli+}.
 
       .. procedure::
 
-         .. include:: /includes/steps-atlas-cli-auth-nonprog-default.rst
+         .. step:: Run the authentication command with the ``--skipConfig`` flag.
+
+            Run the ``atlas auth login`` command in your terminal with the flag
+            to skip profile setup.
+
+            .. code-block:: sh
+
+               atlas auth login --skipConfig
+
+            The command opens a browser window and returns a one-time
+            activation code. This code expires after 10 minutes.
```

Comment:
mh... I have the impression that we are suggesting running the login with `skipConfig`. I would keep the example without the flag and then add a sub-step saying that they can use `skipConfig` if they don't want to set the org/proj Ids

Reply:
@andreaangiolillo Currently we have two step-by-step procedures for logging in: [Connect with minimum required settings ](https://docs-mongodbcom-staging.corp.mongodb.com/atlas-cli/docsworker-xlarge/DOCSP-21720/connect-atlas-cli/#connect-with-minimum-required-settings)and [Create a profile](https://docs-mongodbcom-staging.corp.mongodb.com/atlas-cli/docsworker-xlarge/DOCSP-21720/atlas-cli-save-connection-settings/#create-a-profile). This was by design to give users an option to log in quickly without spending time setting up a profile at all if they chose to do so. Using --skipConfig would be the minimum required settings unless this is not allowed for first-time login, so I think it's appropriate to have this as the command on this page. We do specify on [the tabs here](https://docs-mongodbcom-staging.corp.mongodb.com/atlas-cli/docsworker-xlarge/DOCSP-21720/connect-atlas-cli/#select-a-connection-method) that they have both options for login.

Reply:
Ack. Thanks for the clarification!